_test/test_transformation/test_aProjection_methods.m

```diff
@@ -26,38 +26,41 @@ function test_serialize_deserialize(~)
             ap = aProjectionTester();
             ser_str = ap.serialize();
             ap_rec = serializable.deserialize(ser_str);
-            
+
             assertEqual(ap,ap_rec);
         end
-        function test_constructor_some_fields_and_extra_throws(~)
+        function test_constructor_some_fields_and_extra_returned(~)
             par = {'offset', [1,0,0,0],'lab2','b','extra','blabla','extra1_no_value'};
-            assertExceptionThrown(@()aProjectionTester(par{:}),...
-                'HORACE:aProjection:invalid_argument');
+            ap = aProjectionTester();
+            [ap,extra_par] = ap.init(par{:});
+            assertEqual(ap.label ,{'Q_h','b','Q_l','En'});
+            assertEqual(ap.offset ,[1,0,0,0]);
+            assertEqual(extra_par,{'extra','blabla','extra1_no_value'});
```

Comment:
`ap.extra_par`?

Reply:
ap is not accepting extra_par. This is the point of the approach.